01_BLAST/blast.R

```diff
@@ -0,0 +1,33 @@
+##################################################
+### BLAST sequence alignment implementation
+##################################################
+### Given a specified gene and genome, this code
+### identifies the most likely matching locations
+### in the target genome.
+##################################################
+
+# setup
+if (!requireNamespace("BiocManager", quietly = TRUE)) {
+  install.packages("BiocManager")
+}
+
+BiocManager::install(version="3.19")
+BiocManager::install("rBLAST")
```

Comment:
I like that you use the namespace::export format here. It feels much more readable, in my opinion.